torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2791,6 +2791,13 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    ),
                    sample_kwargs=sample_kwargs_clamp,
                    sample_inputs_func=sample_inputs_clamp),
+    UnaryUfuncInfo('clone',
```

Comment:
Thank you!

Reply:
Adding an OpInfo for clone is a smart way to test positive.

Would you also remove clone's old method_tests() entries:

https://github.com/pytorch/pytorch/blob/6c6ccf48aa40e4d372ea4065af636935a974a4bf/torch/testing/_internal/common_methods_invocations.py#L4698

and create a custom sample input function for the clone opinfo that creates those two cases. Let me know if you have questions about how to do this. Many example ports of method_tests() to OpInfos can be found here https://github.com/pytorch/pytorch/issues/54261. A relatively simple example is https://github.com/pytorch/pytorch/pull/54798.

Reply:
Update!

After looking at the test failures it might be easier to not alias positive to clone. By deleting the alias assignment in the jit, this OpInfo can be changed to:

```
UnaryUfuncInfo('positive',
ref=np.positive,
dtypes=all_types_and_complex_and(torch.half, torch.bfloat16),
dtypesIfCPU=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16), dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
supports_out=False,),
```

With this change I think the tests will pass.